aidants_connect_habilitation/models.py

```diff
@@ -1 +1,145 @@
+    siret = models.BigIntegerField("N° SIRET", default=1)
+    address = models.TextField("Adresse", default="No address provided")
+    zipcode = models.CharField("Code Postal", max_length=10, default="0")
+    city = models.CharField("Ville", max_length=255, null=True)
+
+    partner_administration = models.CharField(
+        "Administration partenaire",
+        blank=True,
+        null=True,
+        default=None,
+        max_length=200,
+    )
+
+    public_service_delegation_attestation = models.FileField(
+        "Attestation de délégation de service public"
+    )
+
+    france_service_label = models.BooleanField(
+        "Labellisation France Services", default=False
+    )
+
+    web_site = models.URLField("Site web")
+
+    # max_lengh?
+    mission_description = models.TextField("Description des missions de la structure")
+
+    avg_nb_demarches = models.IntegerField(
+        "Nombre moyen de démarches ou de dossiers traités par semaine"
+    )
+
+    # Manager
+    manager_first_name = models.CharField("Prénom", max_length=150)
+    manager_last_name = models.CharField("Nom", max_length=150)
+    manager_email = models.EmailField(max_length=150)
+    manager_profession = models.CharField(
+        "Intitulé du poste", blank=False, max_length=150
+    )
+
+    # Checkboxes
+    cgu = models.BooleanField("J'accepte les CGU")
+    dpo = models.BooleanField("Le DPO est informé")
+    aidants_only = models.BooleanField(
+        "La structure ne contient que des aidants professionnels"
+    )
+    elected_free = models.BooleanField("Aucun élu n'est impliqué dans la structure")
+
+    class Meta:
+        constraints = [
+            models.CheckConstraint(
+                check=(
+                    ~Q(type=RequestOriginConstants.OTHER.name)
+                    | (
+                        Q(type=RequestOriginConstants.OTHER.name)
+                        & Q(type_other__isnull_or_blank=False)
+                    )
+                ),
+                name="type_other_correctly_set",
+            ),
+            models.CheckConstraint(check=Q(cgu=True), name="cgu_checked"),
+            models.CheckConstraint(check=Q(dpo=True), name="dpo_checked"),
+            models.CheckConstraint(
+                check=Q(aidants_only=True), name="aidants_only_checked"
+            ),
+            models.CheckConstraint(
+                check=Q(elected_free=True), name="elected_free_checked"
+            ),
+        ]
+
+
+class AidantRequest(models.Model):
+    organisation = models.ForeignKey(
+        OrganisationRequest,
+        null=True,
+        on_delete=models.CASCADE,
+        related_name="aidant_requests",
+    )
+
+    first_name = models.CharField("Prénom", max_length=150)
+    last_name = models.CharField("Nom", max_length=150)
+    email = models.EmailField(max_length=150)
+    profession = models.CharField("Intitulé du poste", blank=False, max_length=150)
+
+
+class RequestMessages(models.Model):
```

Comment:
Un seul message par instance de cette classe, non ?

```suggestion
class RequestMessage(models.Model):
```